workbench/tests/renderer/investtab.test.js

```diff
@@ -227,7 +227,7 @@ describe('Sidebar Buttons', () => {
     const response = 'saved';
     archiveDatastack.mockImplementation(() => new Promise(
       (resolve) => {
-        setTimeout(() => resolve(response), 1000);
+        setTimeout(() => resolve(response), 500);

```

Comment:
This test failed once intermittently. I think the `waitFor` at line 245 times out after 1000ms by default, and it was trying to wait for this to resolve, so it was hit/miss.